services/net/content/ContentManager.cs

```diff
@@ -446,4 +482,47 @@ private async Task<ContentModel> CopyFileWithSSHAsync(DataLocationModel dataLoca
         return result;
     }
     #endregion
+
+    #region Helper Methods
+    private static ContentTonePoolModel? GetTonePoolMapping(IEnumerable<API.Areas.Editor.Models.TonePool.TonePoolModel> tonePools, int toneValue, string? userIdentifier)
+    {
+        // use the "Default" TonePool
+        if (string.IsNullOrEmpty(userIdentifier)) {
+            // the "Default" tone should always be present
+            var tonePoolDefault = tonePools.Where(s => s.Name.Equals("Default")).First();
```

Comment:
Ideally this would be based on configuration rather than hardcoded.

Reply:
fixed by adding a `ContentMigrationOptions` section for the `ContentManager` service.